collectors/meta_template/schema.json

```diff
@@ -0,0 +1,387 @@
+      }
+    },
+    "alerts": {
+      "type": "array",
+      "description": "This section is autogenerated, only user validation is to ensure the \"os\" field is in line with the OS the collector can run on. If this runs on freebsd but the \"os\" field is \"linux\", just delete the list item.",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "link": {
+            "type": "string"
+          },
+          "metric": {
+            "type": "string"
+          },
+          "info": {
+            "type": "string"
+          },
+          "os": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "metrics": {
+      "type": "object",
+      "description": "Autogenerated section,  user input is needed to provide a description for labels if any, and to add the scope's rendered name in each scope.",
+      "properties": {
+        "text": {
+          "type": "string"
+        },
+        "folding": {
+          "type": "object",
+          "description": "Controls whether or not to fold this section, in case the table is very large and we don't need to present it",
+          "properties": {
+            "title": {
+              "type": "string",
+              "description": "Title for the collapsed section"
+            },
+            "enabled": {
+              "type": "boolean",
+              "description": "Boolean value, controls whether to collapse or not"
+            }
+          }
+        },
+        "scopes": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "rendered_name": {
+                "type": "string",
+                "description": "Please fill this in, it should fit:\n\"Metrics per X\" and \"X Level\" where X is the scope name."
+              },
+              "text": {
+                "type": "string",
+                "description": "Description for the scope."
+              },
+              "labels": {
+                "type": "array",
+                "items": {
+                  "name": {
+                    "type": "string"
+                  },
+                  "text": {
+                    "type": "string"
+                }
+              },
+              "metrics": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "name": {
+                      "type": "string"
+                    },
+                    "text": {
+                      "type": "string"
+                    },
+                    "unit": {
+                      "type": "string"
+                    },
+                    "dimensions": {
+                      "type": "array",
+                      "items": {
+                        "type": "string"
```

Comment:
item type is object see [apache meta](https://github.com/netdata/go.d.plugin/blob/b2b7637c59ad51f98eaec93285890fb6d7b76900/modules/apache/metadata.yaml#L226-L227)

Reply:
I had changed this due to not fitting yesterday's yaml schema validator, will change it back :+1: